tests/pytests/test_scorers.py

```diff
@@ -89,12 +89,15 @@ def testDocscoreScorerExplanation(env):
     env.assertEqual(res[8][1], "Document's score is 0.10")
 
 def testTFIDFScorerExplanation(env):
+    conn = getConnectionByEnv(env)
     env.expect('ft.create', 'idx', 'ON', 'HASH', 'SCORE_FIELD', '__score',
                'schema', 'title', 'text', 'weight', 10, 'body', 'text').ok()
     waitForIndex(env, 'idx')
-    env.expect('ft.add', 'idx', 'doc1', 0.5, 'fields', 'title', 'hello world',' body', 'lorem ist ipsum').ok()
-    env.expect('ft.add', 'idx', 'doc2', 1, 'fields', 'title', 'hello another world',' body', 'lorem ist ipsum lorem lorem').ok()
-    env.expect('ft.add', 'idx', 'doc3', 0.1, 'fields', 'title', 'hello yet another world',' body', 'lorem ist ipsum lorem lorem').ok()
+
+    conn.execute_command('ft.add', 'idx', 'doc1', 0.5, 'fields', 'title', 'hello world',' body', 'lorem ist ipsum')
```

Comment:
Can still use `assertCmdOk` or `assertOk` even if not using `expect`
```suggestion
    conn.assertCmdOk('ft.add', 'idx', 'doc1', 0.5, 'fields', 'title', 'hello world',' body', 'lorem ist ipsum')
```

Reply:
@oshadmi I'm driving it from master. The point is to use `conn = getConnectionByEnv(env)` to allow running on cluster. If one uses a valid Connection method it should hold.